tests/README.md

```diff
@@ -0,0 +1,31 @@
+## Frontend Automation Folder Structure
```

Comment:
usually readme starts with name of the directory as a main heading
```
Frontend Tests

Folder structure
```

Reply:
Modified the heading. Thanks!